src/Maker/Factory/FactoryGenerator.php

```diff
@@ -32,6 +32,9 @@ final class FactoryGenerator
     public const PHPSTAN_PATH = '/vendor/phpstan/phpstan/phpstan';
     public const PSALM_PATH = '/vendor/vimeo/psalm/psalm';
 
+    /** @var string[]|true */
+    private array|bool $forceProperties = [];
```

Comment:
`always_force_properties` is [a bool which can be true/false](https://github.com/zenstruck/foundry/blob/2.x/src/ZenstruckFoundryBundle.php#L81-L84). You cannot globally chose to force some specific properties.

please, add this to the constructor, using property promotion, and with `false` as default. We won't use the `methodCall`/`returnsClone` stuff her